msteams-platform/tutorials/UgradeTeamsUsingYeoman.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade teams using yeoman generator
+This article helps you to upgrade teams from current version to the latest version
+## Get current version of teams
+```
+ yo teams --version
+```
+## Update teams
+The yo command lists out various options ranging from creating project to upate generator , use the below command to select updtae generator
+```
+ yo
+```
+Use the arrow keys to choose Update Generators
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/YoSelectUpdateGen.png)
+
+This should listdown all the generators available , you should select a specific generator by using space bar to select and deselect teams from the available options
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams//UseSpaceToSelectGenerators.png)
+
+It takes a while to install teams, may be a few seconds to minutes
+
+Once the installation is complete , you can use the below command to check the version
+
+```
+yo teams --version
+```
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/FindVersionAfterInstallation.png)
```

Comment:
Kindly check and correct file paths, as they should not be pointing to personal folders.

Reply:
@KirtiPereira  Sure as the folder is  approved I will add it.

Also need help in understanding this.
![image](https://user-images.githubusercontent.com/9211327/108087801-c9a2d400-709d-11eb-8f47-71a216df68d6.png)

Reply:
For the validation warnings, you can refer to another document to get an idea.
For example:
Title : Upgrade Teams using Yeoman
Description: This document provides steps on how to upgrade Teams using Yeoman.
For the duplicate-alt-text message change the description little for each of the line mentioned.

Hope this helps.